openedx/core/djangoapps/theming/management/commands/create_sites_and_configurations.py

```diff
@@ -0,0 +1,180 @@
+"""
+This command will be run by an ansible script.
+"""
+
+import json
+import logging
+
+from provider.oauth2.models import Client
+from provider.constants import CONFIDENTIAL
+from edx_oauth2_provider.models import TrustedClient
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+from openedx.core.djangoapps.theming.models import SiteTheme
+
+LOG = logging.getLogger(__name__)
+
+SITES = {
+    "mitxpro": "mitxpro.mit.edu",
+    "hms": "globalacademy.hms.harvard.edu",
+    "edx": "edx-theme"
+}
```

Comment:
please add wharton too

Reply:
actually, would be better to remove this hardcoded dict completely and generate it dynamically by iterating over the theme dirs looking for themes that have a sandbox_configuration.json file